.github/workflows/release-build-and-push-to-GH-releases.yml

```diff
@@ -78,17 +78,3 @@ jobs:
           files: "./dist/channels/**/chectl-*.gz"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Create failure MM message
```

Comment:
you decided to remove this whole notification?

Reply:
could use `if [[ ${GITHUB_REF#refs/heads/} =~ 7.* ]]; then ...` to trap for commits to .x branch (or just not bother since we have the PR release step notifying already)